.claude/dashboard/stack.md

```
@@ -0,0 +1,100 @@
+| `@tanstack/ai` | AI abstractions |
+| `@tanstack/ai-anthropic` | Claude provider |
+| `@tanstack/ai-openai` | OpenAI provider |
+| `@tanstack/ai-gemini` | Gemini provider |
+| `@tanstack/ai-ollama` | Ollama provider |
+| `@tanstack/ai-react` | React hooks for AI |
+
+### UI & Styling
+
+| Package | Purpose |
+|---------|---------|
+| `tailwindcss` | ^4.0.6 | Utility CSS |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)